README.md

```diff
@@ -358,6 +361,7 @@ Configuration settings that have been copied from vim. Vim settings are loaded i
 | vim.textwidth    | Width to word-wrap when using `gq`                                                                                                                                                                                                                                    | Number  | 80            |
 | vim.timeout      | Timeout in milliseconds for remapped commands                                                                                                                                                                                                                         | Number  | 1000          |
 | vim.whichwrap    | Controls wrapping at beginning and end of line. Comma-separated set of keys that should wrap to next/previous line. Arrow keys are represented by `[` and `]` in insert mode, `<` and `>` in normal and visual mode. To wrap "everything", set this to `h,l,<,>,[,]`. | String  | ``            |
+| vim.report       | Threshold for reporting number of lines changed.                                                                                                                                                                                                                      | Number  | 2             |
```

Comment:
maybe `vim.statusline.reportLinesChangedThreshold`?

Reply:
This is a direct translation from vim though http://vimdoc.sourceforge.net/htmldoc/options.html#'report'

Reply:
Oh my, I did not know this existed. Ok, nevermind :)